Gemfile.bk

```diff
@@ -0,0 +1,68 @@
+source 'https://rubygems.org'
+
+ruby File.read('.ruby-version').strip
+
+gem 'rails', '~> 4.2'
+gem 'rails-i18n'
+gem "rdiscount"
+gem 'activeadmin', '~> 1.2.1'
+gem 'has_scope'
+gem 'pundit', '~> 2.0.0'
+gem 'pg', '0.17.1'
+gem 'hstore_translate'
+gem 'dalli'
+gem 'devise', '~> 4.5.0'
+gem "http_accept_language", '~> 2.1.1'
+gem 'unicorn'
+gem 'kaminari', '~> 1.1.1'
+gem "simple_form", ">= 3.0.0"
+gem 'rollbar', '2.8.3'
+gem 'whenever', require: false
+gem 'prawn', '~> 2.2.0'
+gem 'prawn-table', '~> 0.2.2'
+gem 'elasticsearch-model'
+gem 'elasticsearch-rails'
+gem 'skylight'
+gem 'sidekiq', '5.1.3'
+gem 'sidekiq-cron', '0.6.3'
+# TODO: remove this once the following issue has been addressed
+#       https://github.com/ondrejbartas/sidekiq-cron/issues/199
+gem 'rufus-scheduler', '~> 3.4.2'
+
+# Assets
+gem 'jquery-rails', '>= 4.2.0'
+gem 'bootstrap-sass'
+gem 'sass-rails', '~> 5.0.7'
+gem 'coffee-rails'
+gem 'uglifier', '2.7.2'
+gem 'select2-rails'
+
+group :development do
+  gem "binding_of_caller", '~> 0.8.0'
+  gem "better_errors", '~> 2.4.0'
+  gem 'rubocop', '~> 0.52.1', require: false
+  gem 'web-console', '2.1.3'
+  gem 'capistrano', '~> 3.1'
+  gem 'capistrano-rails', '~> 1.1'
+  gem 'capistrano-rbenv', '~> 2.1'
+  gem 'airbrussh', require: false
+  gem 'localeapp', '2.1.1', require: false
+  gem 'letter_opener', '1.4.1'
+  gem 'dotenv-rails', '~> 2.7.1'
+end
+
+group :development, :test do
+  gem "byebug"
+end
+
+group :test do
+  gem "rspec-rails", '~> 3.8.2'
+  gem "database_cleaner", '1.6.2'
+  gem 'shoulda-matchers', '~> 3.1.2'
+  gem 'fabrication'
+  gem 'faker', '~> 1.9'
+  gem 'capybara', '~> 3.13'
+  gem 'selenium-webdriver', '~> 3.141'
+  gem 'webdrivers', '~> 3.0'
+  gem 'simplecov', '~> 0.16.1', require: false
+end
```

Comment:
What is this file?

Reply:
sorry, I'll delete from the repo. It's my gemfile, the provided with timeoverflow doesn'twork, as I explained in https://community.coopdevs.org/t/error-a-lintentar-instalar-timeoverflow-en-local/935

Reply:
It should work. The error related to chromedriver that you mention in that post has been solved by https://github.com/coopdevs/timeoverflow-provisioning/pull/66

Are you using the latest version of https://github.com/coopdevs/timeoverflow-provisioning to create the container?

Reply:
Yes, I'm using the last version of provisioning, but more errors appeared later: 

Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    rails (~> 4.2) was resolved to 4.2.11.1, which depends on
      bundler (< 2.0, >= 1.3.0)

  Current Bundler version:
    bundler (2.0.1)
This Gemfile requires a different version of Bundler.

More details at:
https://community.coopdevs.org/t/error-a-lintentar-instalar-timeoverflow-en-local/935/9?u=zarpilla